packages/fxa-settings/README.md

```diff
@@ -218,12 +218,25 @@ yarn test -t="renders as expected"
 ```
 
 You can also see the test coverage with details by running the following command:
+
 ```
 CI=yes yarn test --coverage --verbose
 ```
 
 Refer to Jest's [CLI documentation](https://jestjs.io/docs/en/cli) for more advanced test configuration.
 
+### Components that use `useAccount`
```

Comment:
Thanks for writing this, I like that it'll come up if someone searches `fxa-settings` for `MockedProvider`. It could go above the "Testing" section I think but it doesn't really matter.